core/src/main/java/hudson/cli/ReloadJobCommand.java

```diff
@@ -78,11 +79,10 @@ protected int run() throws Exception {
                 }
 
                 if(job == null) {
-                    // TODO: JENKINS-30785
-                    AbstractProject project = AbstractProject.findNearest(job_s);
+                    AbstractItem project = Items.findNearest(AbstractItem.class, job_s, jenkins);
```

Comment:
Why not just item? Or maybe even TopLevelItem (who would reload other items anyway?)